Improve Quickstart page (#9048)
jekyll/jekyll: docs/_docs/index.md

favorite markup language and uses layouts to create a static website. You can
tweak the site's look and feel, URLs, the data displayed on the page, and more. 

## Setup

1. Install all [prerequisites]({{ '/docs/installation/' | relative_url }}) (basicly Ruby **{{ site.data.ruby.min_version }}** or higher, GCC and Make.
2. Install the jekyll and bundler [gems]({{ '/docs/ruby-101/#gems' | relative_url }}).
```sh
gem install jekyll bundler
```

## Create a site

1. Create a new Jekyll site at `./my-awesome-site`.
```sh

> jekyll new my-awesome-site

Reviewer: I don't see the need to change `myblog` to something else.
Though the only issue I have with `my-awesome-site` is that it's a *longer* name..

```

> 2. Change into your new directory.

Reviewer: Lets change this to `Move into..`
```suggestion
2. Move into your new directory.
```

```sh
cd my-awesome-site
```

> Jekyll automatically creates several files and directories to help start your site faster.

Reviewer: ```suggestion
Jekyll automatically creates several files and directories to help start working on your site faster.
```

> 3. Edit file `index.markdown` in `./my-awesome-site`, replacing its content with:
> ```markdown
> ---
> layout: home
> title: My really awesome site!
> ---
> # My really awesome site
> 
> Lots of awesome things!
> ```

Reviewer: This step shouldn't be necessary for a *quickstart* guide.
```suggestion
```

## Check the result

Build the site and make it available on a local server.
```sh
bundle exec jekyll serve
```

Now, browse to [http://localhost:4000](http://localhost:4000){:target="_blank"}

## Use version control (Git)

You can initialize a Git repository on your site's directory. 

> One of the great things about Jekyll is there’s no database. All content and site structure are files that a Git repository can version. Using a repository is optional but is recommended. You can learn more about using Git by reading the Git Handbook.

Reviewer: This line is a bit too long. Consider limiting line-lengths to 120 characters.
And since you're mentioning Git Handbook here, you must link to it.

```suggestion
One of the great things about Jekyll is that there's no database. All content and structure are files that a Git repository
can version. Using a repository is optional but recommended. You can learn more about using Git by reading the Git Handbook.
```
